Duplicati/Library/Backend/S3/S3Backend.cs

```diff
@@ -66,6 +66,7 @@ public class S3 : IBackend, IStreamingBackend, IRenameEnabledBackend
             { "Infomaniak Swiss Backup cluster 2", "s3.swiss-backup02.infomaniak.com" },
             { "Infomaniak Swiss Backup cluster 3", "s3.swiss-backup03.infomaniak.com" },
             { "Infomaniak Public Cloud 1", "s3.pub1.infomaniak.cloud" },
+            { "さくらのクラウド (Sakura Cloud)", "isk01.sakurastorage.jp" }

```

Comment:
According to example commands on the page https://manual.sakura.ad.jp/cloud/objectstorage/about.html, the endpoint URL seems to actually be `s3.isk01.sakurastorage.jp`.

It claims so on the actual UI as well:

![1](https://github.com/user-attachments/assets/a4783445-c0b3-43b9-8d85-88a66205f1fe)